aws-lambda-java-log4j2/pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.amazonaws</groupId>
     <artifactId>aws-lambda-java-log4j2</artifactId>
-    <version>1.6.0</version>
+    <version>1.6.1</version>
```

Comment:
regarding semantic versioning, since we update the minor version of `log4j` we may want to bump the minor version of our artifact? WDYT?

Reply:
changed, makes sense